.github/workflows/bump_stdlibs.yml

```diff
@@ -0,0 +1,31 @@
+name: bump_stdlibs
+
+on:
+  schedule:
+    - cron: '0 0 * * 1'
+
+jobs:
+  bump_stdlibs:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.3.0]
+        julia-arch: [x86]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@af513c7a016048ae468971c52ed77d9562c7c819 # v1.0.0
+      - uses: julia-actions/setup-julia@082493e5c5d32c1fa68c35556429b0f1b2807453 # v1.0.1
+        with:
+          version: ${{ matrix.julia-version }}
+      - name: Install dependencies
+        run: julia -e 'import Pkg; Pkg.add(["GitCommand", "GitHub"])'
```

Comment:
Should use a manifest and instantiate so this uses constant versions. 

Also why is  GitCommand needed? I think requiring that git is installed is totally fine.

Reply:
Good idea, I’ll add a manifest.

I can also remove GitCommand and use local git instead.